lib/docx/document.rb

```diff
@@ -30,8 +30,7 @@ def initialize(path_or_io, options = {})
         @zip = Zip::File.open_buffer(path_or_io)
       end
 
-      document = @zip.find_entry('word/document.xml')
-      document ||= @zip.find_entry('word/document2.xml')
+      document = @zip.glob('word/document*.xml').first
```

Comment:
I don't know the specification of Office Open XML file format. 
Is there no docx file which have some xml files with prefix `'word/document'`?
The PR #73 tries to deal with all the xml.

Reply:
The specifications can be found here: http://officeopenxml.com/anatomyofOOXML.php

---

If we want to be more clean, we should load `[Content_Types].xml` and check the name of the document.

I will work on a new PR that will use `[Content_Types].xml` to define all XML.